lib/index.js

```diff
@@ -19,9 +20,10 @@ module.exports = curry['default'] = curry
 function curry (fn) {
   var self = this
   return function () {
-    var args1 = slice(arguments)
+    var args = slice(arguments)
```

Comment:
sounds good. i'll try to make a point of it as well ... especially in the really low level micros.